table_detection/table_increment[] increment_arr